Runtime/LLMCharacter.cs

```diff
@@ -137,6 +137,8 @@ public class LLMCharacter : LLMCaller
         protected SemaphoreSlim chatLock = new SemaphoreSlim(1, 1);
         protected string chatTemplate;
         protected ChatTemplate template = null;
+        protected Task grammarTask;
```

Comment:
Great implementation for grammar! This is not needed anymore in release v3.0.0 because I load the grammar from the file directly and just keep it as a serialized string